src/api/users/docs/FAQ.md

```diff
@@ -0,0 +1,20 @@
+# FAQ
+
+This document explains several frequently answered questions related to the Users Micro-Service and Firestore. See [`README.md`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/users/README.md) for more information.
```

Comment:
I'm not an English expert, but isn't FAQ stands for frequently _asked_ questions?